crates/aptos/src/move_tool/show.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use super::IncludedArtifactsArgs;
+use crate::common::types::{CliCommand, CliError, CliResult, CliTypedResult, MovePackageDir};
+use anyhow::Context;
+use aptos_framework::{BuildOptions, BuiltPackage};
+use aptos_types::transaction::EntryABI;
+use async_trait::async_trait;
+use clap::{Parser, Subcommand};
+
+/// Commands for showing information about compilation artifacts.
+#[derive(Subcommand)]
+pub enum ShowTool {
+    /// Compile the package and show information about the ABIs of the compiled modules.
```

Comment:
Don't need this comment here, the Abi tool will be used for the CLI help

Reply:
Neato.